unit-tests/algo/d2rgb/test-frame-drop.py

```diff
@@ -0,0 +1,88 @@
+import pyrealsense2 as rs, common as test, ac
```

Comment:
If we're always going to be doing `common as test` -- maybe we should just rename it `test`?

Reply:
I thought it was most readable like this. The file is a common place to put things that are used for all tests, but in the test common.start seems less readable then test.start. If you want me to change it I will

Reply:
Let's go for it

Reply:
done